presto-orc/src/main/java/com/facebook/presto/orc/metadata/OrcMetadataWriter.java

```diff
@@ -311,6 +312,10 @@ public int writeStripeFooter(SliceOutput output, StripeFooter footer)
 
     private static OrcProto.ColumnEncoding toColumnEncoding(ColumnEncoding columnEncodings)
     {
+        Preconditions.checkArgument(
```

Comment:
This assumes that only DWRF_MAP_FLAT columns have ColumnEncodings with non-zero sequenceIds.  Logically, for now, that's a reasonable assumption, but I was worried that that may change in the future and, more importantly, there's nothing in the code that enforces that.  I thought it was better to directly check that there are no SequenceEncodings (since I don't want to accidentally ignore those) rather than check the column encoding and rely on that assumed relation between the two.